exchanges/gateio/gateio_test.go

```diff
@@ -1450,8 +1446,12 @@ func TestCancelAllFuturesOpenOrders(t *testing.T) {
 
 func TestGetAllDeliveryContracts(t *testing.T) {
 	t.Parallel()
-	_, err := g.GetAllDeliveryContracts(context.Background(), currency.USDT)
-	assert.NoError(t, err, "GetAllDeliveryContracts should not error")
+	r, err := g.GetAllDeliveryContracts(context.Background(), currency.USDT)
+	require.NoError(t, err, "GetAllDeliveryContracts must not error")
+	assert.NotEmpty(t, r, "GetAllDeliveryContracts should return data")
+	r, err = g.GetAllDeliveryContracts(context.Background(), currency.BTC)
+	require.NoError(t, err, "GetAllDeliveryContracts must not error")
+	assert.Empty(t, r, "GetAllDeliveryContracts shouldn't return data")
```

Comment:
Requires better explanation:
` ... in some context`
In this case probably `should not return data with unsupported currency BTC`

Also: Don't think we should/can abbreviate should not to `shouldn't` under normal [rfc2119](https://www.rfc-editor.org/rfc/rfc2119) ( which is what I'm always aiming for )